src/Native/System.Security.Cryptography.Native/pal_evp_cipher.h

```diff
@@ -126,3 +126,20 @@ EvpDes3Cbc
 Direct shim to EVP_des_ede3_cbc.
 */
 extern "C" const EVP_CIPHER* CryptoNative_EvpDes3Cbc();
+
+/*
+Function:
+EvpRc4
+
+Direct shim to EVP_rc4.
+*/
+extern "C" const EVP_CIPHER* CryptoNative_EvpRc4();
+
+/*
+Function:
+EvpCipher
+
+Direct shim to EVP_Cipher.
+*/
+extern "C"
+void CryptoNative_EvpCipher(EVP_CIPHER_CTX* ctx, uint8_t *out, const uint8_t *in, uint32_t inl);
```

Comment:
Is this really any different than the series CipherInit, CipherUpdate, CipherFinal; all of which are already exposed?